api/src/test/java/com/epam/pipeline/test/acl/AclTestConfiguration.java

```diff
@@ -37,10 +39,18 @@
  */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
+@ContextConfiguration(classes = {AclTestBeans.class, AclSecurityConfiguration.class})
+@ComponentScan(
```

Comment:
It seems that you can just use `basePackages = {"com.epam.pipeline.security.acl", "com.epam.pipeline.manager.security"}` without excludes